client/state/sites/domains/selectors.js

```diff
@@ -79,22 +78,5 @@ export const isUpdatingDomainPrivacy = ( state, siteId, domain ) => {
  */
 export const getDecoratedSiteDomains = treeSelect(
 	( state, siteId ) => [ getDomainsBySiteId( state, siteId ) ],
-	( [ domains ] ) => {
-		if ( ! domains ) {
-			return null;
-		}
-
-		return domains.map( domain => ( {
-			...domain,
-			autoRenewalMoment: domain.autoRenewalDate ? moment( domain.autoRenewalDate ) : null,
-			registrationMoment: domain.registrationDate ? moment( domain.registrationDate ) : null,
-			expirationMoment: domain.expiry ? moment( domain.expiry ) : null,
-			transferAwayEligibleAtMoment: domain.transferAwayEligibleAt
-				? moment( domain.transferAwayEligibleAt )
-				: null,
-			transferEndDateMoment: domain.transferStartDate
-				? moment( domain.transferStartDate ).add( 7, 'days' )
-				: null,
-		} ) );
-	}
+	( [ domains ] ) => domains || null
```

Comment:
We no longer need `treeSelect` here, as we're not transforming the domains. And the `Decorated` in the selector name is no longer appropriate either.

Reply:
Removed `getDecoratedSiteDomains` altogether, as it's now effectively just an alias of `getDomainsBySiteId`.